VERSION

```diff
@@ -0,0 +1 @@
+1.0.0-pre1
```

Comment:
> Agree we should capture the service version external to the vector-datasource repo. Should that go into a new repo and/or one of the chef recipes?

It would probably be easiest to generate a json file in a recipe and push that over to s3. We can read the revisions of the software components from our stack settings. And we can move the version there as well, or we can just read this new version file from the tagged vector-datasource if you think it would be easier for you to manage that way. We have a step that generates the tilejson files already, so we can probably update/rename that step to generate this as well for a particular deploy.